src/domUpdates.js

```diff
@@ -430,7 +433,21 @@ let domUpdates = {
   displayMessageButtons(e) {
     const buttonMessage = document.getElementById(`m${e.target.id}`);
     buttonMessage.innerText = postMessage;
-    setTimeout(() => {buttonMessage.classList.add('fade-out')}, 2000);
+    setTimeout(() => {
+      buttonMessage.classList.add('fade-out')
+    }, 2000);
+  },
+
+  disableButtons() {
+    const minusButtons = document.querySelectorAll('.minus-js');
+    minusButtons.forEach(button => {
+      const ingredient = currentUser.pantry.ingredients.find(item => {
+        return item.ingredient === parseInt(button.id)
+      })
+      if (ingredient.amount === 0) {
+        button.classList.add('disable-button');
+      }
+    })
   },
 
```

Comment:
Great job! Important to have this so people can't subtract from their ingredients if they don't have any (0)